cross/build-rootfs.sh

```diff
@@ -17,10 +17,10 @@ __BuildArch=arm
 __UbuntuArch=armhf
 __UbuntuRepo="http://ports.ubuntu.com/"
 __UbuntuPackagesBase="build-essential libunwind8-dev gettext symlinks liblttng-ust-dev libicu-dev"
-__UbuntuPackages="$__UbuntuPackagesBase"
 if [ -z "$LLVM_ARM_HOME" ]; then
     __LLDB_Package="lldb-3.6-dev"
 fi
+__UbuntuPackages="$__UbuntuPackagesBase $__LLDB_Package"
```

Comment:
Done -- there's actually a larger refactor (including tests, see #7645) that I see as useful: splitting the argument parsing from the logic to set the environment variables correctly.

Would that be better as part of this PR, or a new/separate PR once the specific issue for this PR is fixed?